commonblocks/static/commonblocks/js/hallo-bootstrap.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * registerHalloPlugin() applies to all editor instances of a Page.
+ * We want to encapsulate custom plugin configuration, per field.
+ */
+function makeHalloSimpleRichTextEditable(id, plugins) {
+    var defaultPlugins = $.extend({}, halloPlugins);
+
+    if (typeof plugins === 'object') {
+        $.each(plugins, function(plugin, options) {
+            registerHalloPlugin(plugin, options);
+        });
+
+        makeHalloRichTextEditable(id);
```

Comment:
This, the register function, `halloPlugins`, and the function we are inside of, all rely on this code being loaded directly via a script tag, in the global scope. Would it be possible to more explicitly use `window` as our shared namespace? So `window.registerHalloPlugin`, `window.makeHalloRichTextEditable`, `window.halloPlugins` and finally `window.makeHalloSimpleRichTextEditable = makeHalloSimpleRichTextEditable;`.